### ISSUE TITLE
undefined method `arguments' for ProfilePictureSizes:GraphQL::EnumType

### ISSUE THREAD

**Issue by mvgijssel** [rmosolgo/graphql-ruby#531]
You can get that exception triggered by https://github.com/rmosolgo/graphql-ruby/blob/master/lib/graphql/schema/warden.rb#L103 when doing a query like
```
query {
  me {
    profile_picture(size:{a: b})
  }
}
```
where the profile_picture field is a simple EnumType:
```
ProfilePictureSizeEnum = GraphQL::EnumType.define do
  name 'ProfilePictureSizes'
  description 'Different possible profile picture sizes'

  value 'small', '62x62'
  value 'medium', '82x82'
  value 'large', '110x110'
  value 'xtralarge', '260x260'
end
```

I'd like to make a PR, but not sure what would be the right place to fix this and how to make a generic fix. Should it be in https://github.com/rmosolgo/graphql-ruby/blob/master/lib/graphql/static_validation/rules/arguments_are_defined.rb#L6? Or in warden itself?

**Comment by rmosolgo** [comment]
Hmm, I don't quite understand the root of the issue. Could you share the full stack trace? 

Or, if you'd like to start a PR, how about pushing a branch with a failing test case?

**Comment by mvgijssel** [comment]
I'll push a branch!

**Comment by rmosolgo** [comment]
Turns out, scalars were handled properly but enums weren't! Taking a crack at it in #604

**Comment by rmosolgo** [comment]
This will be fixed in 1.5.2 which I'll release today or tomorrow!

**Comment by mvgijssel** [comment]
> I'll push a branch!

...🙈 

Awesome that you got it fixed!
